src/d_clisrv.c

```diff
@@ -2005,7 +2006,10 @@ static boolean CL_ServerConnectionTicker(boolean viams, const char *tmpsave, tic
 #endif
 	}
 	else
-		I_Sleep();
+	{
+		I_Sleep(cv_sleep.value);
```

Comment:
```suggestion
		if (cv_sleep.value > 0)
			I_Sleep(cv_sleep.value);
```

Same thing here, causes a freeze when connecting to a server. Thinking about it, maybe we should change the defaults of `cv_sleep` instead?